### PR TITLE
NMA-763: DashPay: add hyphen validation for usernames

### PR DESCRIPTION
**Pull request by HashEngineering**
## Issue being fixed or feature implemented

NMA-763

Closer alignment with iOS and Designs.  Includes removing shadow around edit field and some changes to when the checkmark and x's are shown next to the rules.

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Comment by HashEngineering** [comment]
This was tested by disabling checks for dash and platform availability

File: wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java [hunk 1]
```diff
@@ -461,6 +461,9 @@ public void onPreBlockProgressUpdated(PreBlockStage stage) {
             if (stage == PreBlockStage.Starting && lastPreBlockStage == PreBlockStage.None) {
                 startPreBlockPercent = syncPercentage;
             }
+            if (preBlocksWeight > 0.99) {
```
Comment (sambarboza) [inline wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java:464]: Is this related to the ticket?

**Review by sambarboza — APPROVED** [review]

File: wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java:464]: This is not related to the ticket, but does address a cause of having a sync status > 100%.

Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java]: This change was moved to https://github.com/dashevo/dash-wallet/pull/608
